src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/CustomerPreferences/GeneralType.php

```diff
@@ -62,10 +64,11 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     'Send an email with a summary of the account information after registration.',
                     'Admin.Shopparameters.Help'
                 ),
+                'multistore_configuration_key' => 'PS_CUSTOMER_CREATION_EMAIL',
             ])
-            ->add('password_reset_delay', TextWithUnitType::class, [
+            ->add('password_reset_delay', IntegerType::class, [
```

Comment:
Please create a new type `IntegerWithUnitType` instead so we can keep the suffix

Reply:
@atomiix WIP here: https://github.com/PrestaShop/PrestaShop/pull/27788